src/handlers/me.js

```diff
@@ -19,6 +24,39 @@ module.exports = async function me(member, _, message) {
       )
     }
     if (verificationInfo.userVerified) {
+      // Sponsor member
```

Comment:
This flow might need to go somewhere else, because I think that it's not possible to know that a user is verified until they're already sponsored.  In other words, `verification.userVerified` would be `false` if they're not sponsored.